### ISSUE TITLE
Telegram not displayed pictures when external_image_storage is set to local

### ISSUE THREAD

**Issue by Pelt10** [grafana/grafana#24532]
Hello,

**What happened**:
When the external_image_storage option is configured,  a link to the image is given but the image isn't displayed on Telegram.
![image](https://user-images.githubusercontent.com/7582586/81577373-9febac80-93a9-11ea-90c1-ed059fcb2d78.png)

**What you expected to happen**:
Image displayed like when external_image_storage isn't configured.
![image](https://user-images.githubusercontent.com/7582586/81577063-4be0c800-93a9-11ea-861d-6949d5cee522.png)

**How to reproduce it (as minimally and precisely as possible)**:
Just set external_image_storage option to local.

**Environment**:
- Grafana version: Grafana v6.7.3 (a04ef6cefc)
- Data source type & version: Prometheus 2.7.1+ds-3+b11
- OS Grafana is installed on: Debian 10.4
- User OS & Browser: Debian 10.4 & Firefox 68.8.0esr
- Grafana plugins:
    - briangann-gauge-panel @ 0.0.6
    - camptocamp-prometheus-alertmanager-datasource @ 0.0.7
    - grafana-clock-panel @ 1.0.3
    - marcuscalidus-svg-panel @ 0.3.3

Thanks for your help

**Comment by ivanahuckova** [comment]
Hello @Pelt10, could you please share with us your external_image_storage config.

**Comment by Pelt10** [comment]
Hello @ivanahuckova,
My external_image_storage config below: 

```
#################################### External image storage ##########################
[external_image_storage]
# Used for uploading images to public servers so they can be included in slack/email messages.
# you can choose between (s3, webdav, gcs, azure_blob, local)
provider = local

[external_image_storage.s3]
;bucket =
;region =
;path =
;access_key =
;secret_key =

[external_image_storage.webdav]
;url =
;public_url =
;username =
;password =

[external_image_storage.gcs]
;key_file =
;bucket =
;path =

[external_image_storage.azure_blob]
;account_name =
;account_key =
;container_name =

[external_image_storage.local]
# does not require any configuration

[rendering]
# Options to configure external image rendering server like https://github.com/grafana/grafana-image-renderer
;server_url =
;callback_url =

[enterprise]
# Path to a valid Grafana Enterprise license.jwt file
;license_path =

[panels]
;enable_alpha = false
# If set to true Grafana will allow script tags in text panels. Not recommended as it enable XSS vulnerabilities.
;disable_sanitize_html = false
```

**Comment by Pelt10** [comment]
This issue seem to be fix on grafana 7.0.1(ef5b586d7d)
